.azure-pipelines.yml

```diff
@@ -401,7 +374,7 @@ jobs:
     displayName: 'Publish results'
     inputs:
       artifactName: 'edge-stable-results'
-  - template: tools/ci/azure/cleanup_win10.yml
```

Comment:
Can you remove this file? I presume it's unused now.

Reply:
Yes, removed